### PR TITLE
OKD to 40

### PR DESCRIPTION
**Pull request by mburke5678**
Topic Map updates to support moving the OKD docs to 4.3
https://docs.google.com/document/d/1wqaaz_DQ2Y86OfhRyNKMua0TbwlIT27jQUDFLHjgfEQ/edit

### CODE REVIEW

File: _topic_map.yml [hunk 1]
```diff
@@ -34,6 +34,16 @@ Topics:
 - Name: Legal notice
   File: legal-notice
 ---
+Name: What's New?
+Dir: whats_new
+Distros: openshift-origin
+Topics:
+- Name: Overview
+  File: index
+ - Name: Release Notes
```
Comment (vrutkovs) [inline _topic_map.yml]:
```suggestion
  - Name: Release Notes
```
A missing space breaks yaml structure

File: _topic_map.yml [hunk 2]
```diff
@@ -221,16 +232,20 @@ Topics:
...
   File: gathering-cluster-data
-  Distros: openshift-enterprise,openshift-webscale,openshift-dedicated
+  Distros: openshift-enterprise,openshift-webscale,openshift-dedicated,openshift-origin
+- Name: Remote health monitoring with connected clusters
+  Dir: remote_health_monitoring
+  Distros: openshift-enterprise,openshift-webscale,openshift-dedicated,openshift-origin
 - Name: Remote health monitoring with connected clusters
   Dir: remote_health_monitoring
-  Distros: openshift-enterprise,openshift-webscale,openshift-dedicated
+  Distros: openshift-enterprise,openshift-dedicated,,openshift-webscale,openshift-origin
```
Comment (vrutkovs) [inline _topic_map.yml]:
```suggestion
  Distros: openshift-enterprise,openshift-dedicated,openshift-webscale,openshift-origin
```
extra comma  - not sure if this is critical

File: _topic_map.yml [hunk 3]
```diff
@@ -184,6 +191,7 @@ Topics:
   File: installing-gather-logs
 - Name: Support for FIPS cryptography
   File: installing-fips
+  Distros: openshift-enterprise,openshift-dedicated,openshift-online
```
Comment (vikram-redhat) [inline _topic_map.yml]: @mburke5678 openshift-webscale?

File: _topic_map.yml [hunk 4]
```diff
@@ -221,16 +229,17 @@ Topics:
 ---
 Name: Support
 Dir: support
-Distros: openshift-enterprise,openshift-webscale,openshift-online,openshift-dedicated
+Distros: openshift-enterprise,openshift-webscale,openshift-online,openshift-dedicated,openshift-origin
```
Comment (vikram-redhat) [inline _topic_map.yml]: @vrutkovs is this support section valid for origin?

Comment (vrutkovs) [reply _topic_map.yml]: It is, especially "Gathering info"

**Comment by vrutkovs** [comment]
LGTM, @vikram-redhat could you have another look?

**Comment by vikram-redhat** [comment]
@mburke5678 looks good. The only thing left is to change in _distro_map.yml master-3 to master: https://github.com/openshift/openshift-docs/blob/master/_distro_map.yml#L9.

Then it should be good to merge.

**Pull request merged**